PocketBeagle/Grove/Lesson-1.Control-the-Light/README.md

```diff
@@ -0,0 +1,29 @@
+# Description:
```

Comment:
I'd like this to be simply part of Kit.md. The instructions are pretty short and I don't see any reason we can't have them all in one file. Eliminating the directories is especially useful for organizing Python modules.